### DISCUSSION TITLE
Index Web / URL content with links to docs

### DISCUSSION THREAD

**Discussion by jdnuckolls** [microsoft/PubSec-Info-Assistant#634] [Ideas]
Increasingly my customers and peers across US SLG is requesting the ability to **index their public facing website content and associated links to PDF documents**.  While this feature is available (in preview) in Azure AI Studio, it's limited to only 20 pages.
This is not the same as indexing HTML pages (which we already have here), but rather **providing a sitemap or URL and perhaps a # of layers deep to index content, documents, and provide citations that point directly to the web pages**.

**Comment by dayland** [comment]
I believe this is tracked under enhancement #373 . Please add any additional information there if they are the same. 

This enhancement has not yet been prioritized or accepted at this time.

**Comment by jdnuckolls** [comment]
Unfortunately, #373 is not listed under Feature Requests/Ideas.
This is a request that I've had from every customer and I have seen others request it as well.
Perhaps there is a reason this hasn't taken priority that I'm not aware of, but we have to deploy different repos that do have this feature instead and it's a shame because this repo has some great features.


Jeff Nuckolls
Microsoft Corporation
Director - Azure Data/Artificial Intelligence | U.S. State and Local Government
***@***.******@***.***>  | mobile or text: +1 810-877-1873
***@***.***
[A logo of a university  Description automatically generated] [A logo with text and a red ribbon  Description automatically generated] [A blue and white logo with a star  Description automatically generated] [A blue and white logo with a white star  Description automatically generated] [A blue and white logo with a star  Description automatically generated] [A blue and white logo  Description automatically generated] [Microsoft Certified: Azure AI Engineer Associate - Credly]
